core/components/analytics/provider.tsx

```diff
@@ -21,9 +21,6 @@ const getAnalytics = (
   if (settings?.webAnalytics?.ga4?.tagId && channelId) {
     const googleAnalytics = new GoogleAnalyticsProvider({
       gaId: settings.webAnalytics.ga4.tagId,
-      // TODO: Need to implement consent mode
-      // https://github.com/bigcommerce/catalyst/issues/2066
-      consentModeEnabled: false,
```

Comment:
I see you removed this, but we need to account for this in some regions where you don't need to pass consent.